src/app/core/config/config-fix.ts

```diff
@@ -309,6 +309,7 @@ export const defaultJsonConfig = {
       "entity": "School",
       "columns": [
         "name",
+        { id: "DisplayParticipantsCount", viewComponent: "DisplayParticipantsCount", label: "Active Participants" },
```

Comment:
```suggestion
        { id: "DisplayParticipantsCount", viewComponent: "DisplayParticipantsCount", label: $localize`Schools` },
```
I think this makes it a bit more easier to understand in our demo setup. Also adding `i18n`. The translations we normally add as part of our release process.

Reply:
Should be "$localize`Students`", right?